daffodil-cli/src/it/resources/org/apache/daffodil/CLI/testNonCompatibleImplementation.tdml

```diff
@@ -16,10 +16,13 @@
   limitations under the License.
 -->
 
-<tdml:testSuite suiteName="Info" description="CLI"
```

Comment:
The file infoTest.tdml was unused. I copied and edited it, and I guess it was small so we got a diff here instead of a new file and a deleted file.